### ISSUE TITLE
Add Spacer notebook

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#3416]
For completeness I would recommend adding a `Spacer` Reference Notebook at https://panel.holoviz.org/reference/layouts/Spacer.html

Besides the normal reference stuff, it could also show some use cases to make Rows and Columns span and align things.

**Comment by ahuang11** [comment]
Yup still an issue; same as https://github.com/holoviz/panel/issues/3415
